crates/turbo-tasks-fs/src/virtual_fs.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::{bail, Result};
+use turbo_tasks::{primitives::StringVc, CompletionVc, ValueToString, ValueToStringVc};
+
+use super::{
+    DirectoryContentVc, FileContentVc, FileMetaVc, FileSystem, FileSystemPathVc, FileSystemVc,
+    LinkContentVc,
+};
+
+#[turbo_tasks::value]
+pub struct VirtualFileSystem;
+
+#[turbo_tasks::value_impl]
+impl VirtualFileSystemVc {
+    #[turbo_tasks::function]
```

Comment:
```suggestion
```

to force fresh identity on every call of new()

Reply:
Why is that useful?

Reply:
We want virtual file systems to have a different identity. For instance, if you use virtual file systems to represent the Node and Client FS in a test, you'd want them to map to different FS instances for safety, as we have some `FileSystemPathVc` methods that check FS identity in order to determine whether a path operation is correct.